akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
   // no flags currently
 }
 //#snapshot-store-flags
+
+trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
```

Comment:
@VaughnVernon The inmemory plugin also has this 'problem'. For that plugin the solution is quite simple, just mark each event with a unique incrementing ID for example the `java.util.concurrent.atomic.AtomicLong`. For the RDBMS the database itself is ultimately responsible for this. I'm just wondering how the migration script would look like for current users :)